html/templates/settings-payment-method.html

```diff
@@ -1,3 +1,12 @@
+{{#if generalInfoContent}}
+<div class="settings-payment-method-general-info">
+	<h3 class="label">{{generalInfoLabel}}</h3>
+	<div class="content">
+		<p>{{{generalInfoContent}}}</p>
```

Comment:
Avoid triple braces (`{{{..}}}`) because it could expose the app to potential XSS.